Gruntfile.js

```diff
@@ -206,7 +189,7 @@ module.exports = function(grunt) {
     grunt.task.run(["watch"]);
   });
 
-  grunt.registerTask("lint", ["parallelize:tslint", "jscs", "eslint"]);
+  grunt.registerTask("lint", ["jscs", "eslint"]);
```

Comment:
heads up you can also drop eslint completely now (separate PR). tslint supports linting JS files and it's simpler to just have 1 configuration / fewer dependencies